mutiny-core/src/onchain.rs

```diff
@@ -24,12 +25,11 @@ use crate::error::MutinyError;
 use crate::fees::MutinyFeeEstimator;
 use crate::labels::*;
 use crate::logging::MutinyLogger;
-use crate::storage::{MutinyStorage, OnChainStorage};
+use crate::nodemanager::TransactionDetails;
+use crate::storage::{MutinyStorage, OnChainStorage, KEYCHAIN_STORE_KEY, NEED_FULL_SYNC_KEY};
 use crate::utils::{now, sleep};
 
-pub(crate) const DEFAULT_STOP_GAP: usize = 10;
```

Comment:
What's happening with this and what's the stop gap now?

Reply:
The way we normally doesn't really have a stop gap, it just looks at the addresses that we've generated and pending txids, doesn't scan further ahead. BDK now handles this for us so can delete this value

Reply:
How does BDK handle it? Does it not use the stop gap? Want to make sure we don't reintroduce a whole lot of calls if we can help it.

Reply:
Basically it uses the better syncing logic we implemented in #690 but now internally, we only had to have a gap limit because their api use to always require it, now it better supports the more efficient syncing that we were doing